pandas/io/formats/format.py

```diff
@@ -77,6 +77,16 @@
     List[Callable], Tuple[Callable, ...], Mapping[Union[str, int], Callable]
 ]
 FloatFormatType = Union[str, Callable, "EngFormatter"]
+ColspaceType = Union[
+    Mapping[Union[str, int], Union[str, int]],
+]
+ColspaceArgType = Union[
+    str,
+    int,
+    List[Union[str, int]],
```

Comment:
Instead of doing `List` and `Tuple` separately we can just do `Sequence[Union[str, int]]` on one line